docs/contributing/code-style/web/angular.md

```diff
@@ -212,7 +212,7 @@ component "Organization Reports Module" {
 @enduml
 ```
 
-## Reactivity ([ADR-0003](../../../architecture/adr/0003-observable-data-services.md) & ADR-0028)
```

Comment:
Broken reference that was caught as part of this work. There is no ADR 28, this was presumably meant to reference the Signals ADR here: https://github.com/bitwarden/contributing-docs/pull/632 however that has not been merged yet.